### PR TITLE
Added Pull Request Template [WIP]

### PR DESCRIPTION
**Pull request by lfarah**
Pull request template

### CODE REVIEW

**Comment by piv199** [comment]
We need such positions:

- [] Changed more than one extension, but all changes are related
- [] Trivial change (doesn't require changelog)

**Comment by goktugyil** [comment]
Is this going to show while you're sending a PR? Otherwise it might be completely useless

**Comment by piv199** [comment]
@goktugyil otherwise danger will fail

**Comment by lfarah** [comment]
@goktugyil yes, this shows when you're seding a PR. Take a look on how we do on AwesomeiOS:
screen shot 2016-10-30 at 9 49 31 pm

**Comment by lfarah** [comment]
@piv199 done! ✅

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## Checklist
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! 😬 -->
 - [ ] New Extension
 - [ ] New Test
 - [ ] Improvements
```
Comment (piv199) [inline PULL_REQUEST_TEMPLATE.md]: What the purpose of this?

Comment (lfarah) [inline PULL_REQUEST_TEMPLATE.md]: I would say that #228 would be inside ```Improvements```

Comment (lfarah) [inline PULL_REQUEST_TEMPLATE.md]: But we added ```Trivial Change```, so should I remove ```Improvements```?

**Comment by lfarah** [comment]
@piv199 done!

**Comment by lfarah** [comment]
Yaaay It works! 🎉 
screen shot 2016-11-01 at 10 06 18 am

**Pull request merged**